src/Umbraco.Web/Editors/LocalizationController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AutoMapper;
+using System.Collections.Generic;
+using Umbraco.Core.Models;
+using Umbraco.Web.Models.ContentEditing;
+using Umbraco.Web.Mvc;
+using Umbraco.Web.WebApi.Filters;
+using System.Linq;
+
+using Constants = Umbraco.Core.Constants;
+
+namespace Umbraco.Web.Editors
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <remarks>
+    /// 
+    /// </remarks>
+    [PluginController("UmbracoApi")]
+    [UmbracoTreeAuthorize(Constants.Trees.Languages)]
+    [EnableOverrideAuthorization]
+    public class LocalizationController : UmbracoAuthorizedJsonController
```

Comment:
These methods are really only used for the content section and they don't have any security applied to them. Perhaps these should just be part of the content controller since that's the only place they are used? Else section security should be applied to this controller to require access to the content section